.github/workflows/ansible-test-plugins.yml

```diff
@@ -53,16 +53,11 @@ jobs:
           - mariadb
         db_engine_version:
           - '8.0.38'
-          # - '8.4.1'  # Unable to install mysql-client on Ubuntu 20.04 default container used by ansible-test
+          - '8.4.1'
           - '10.4.34'
           - '10.5.25'
           - '10.6.18'
```

Comment:
```suggestion
```
should we remove these intermediate versions and use only the oldest supported and the newest versions we can?

Reply:
Should we then remove 10.4 and 10.6.18 and keep 10.5?

Reply:
the justification for keeping only the oldest supported and the latest is that usually breaking changes introduced in the versions in between are also present in the latest version

Reply:
I like this idea! I'm doing it!